### ISSUE TITLE
Erlangapp clean error

### ISSUE THREAD

**Issue by josephDunne** [goertzenator/cargo-erlangapp#1]
Hi,

I get the following error when I run erlangapp clean in a erlang.mk project:

```
 .cargo-erlangapp/bin/cargo-erlangapp clean
Cleaning /Users/jo/src/flint/flint_hss/crates/auc_nif
Cleaning /Users/jo/src/flint/flint_hss/crates/milenage
Error: can't delete output dir
```

```
› make clean
.cargo-erlangapp/bin/cargo-erlangapp clean 
Cleaning /Users/jo/src/flint/flint_hss/crates/auc_nif
Cleaning /Users/jo/src/flint/flint_hss/crates/milenage
Error: can't delete output dir
make: *** [rust-clean] Error 1
```

I see the app is trying to remove a priv directory somewhere but its swallowing the error so I don't know why its failing. 

Directory structure after make (roughly):

```
.cargo-erlangapp
.cargo-erlangapp/bin/cargo-erlangapp

deps
deps/rust_mk
deps/rust_mk/plugins.mk
deps/rust_mk/....etc

src
src/erlang_code.erl
src/...etc

ebin
ebin/erlang_code.beam

crates
crates/crate1/Cargo.toml
crates/crate1/etc
crates/crate2/Cargo.toml
crates/crate2/etc

priv
priv/crates/crate1/libray.so
priv/crates/crate2/libray.so

_rel/
etc...
```

Directory structure after make clean (with error):

```
.cargo-erlangapp
.cargo-erlangapp/bin/cargo-erlangapp

deps
deps/rust_mk
deps/rust_mk/plugins.mk
deps/rust_mk/....etc

src
src/erlang_code.erl
src/...etc

crates
crates/crate1/Cargo.toml
crates/crate1/etc
crates/crate2/Cargo.toml
crates/crate2/etc

priv

_rel/
etc..
```

i.e. priv directory is there but empty.

**Comment by goertzenator** [comment]
One of the crate tests is currently failing.  Possibly related.

**Comment by goertzenator** [comment]
This should be fixed in today's 0.1.4 release (specifically https://github.com/goertzenator/cargo-erlangapp/commit/377922a7c1c6aa54a4ce791a4d7fe7533a90f6cf), please confirm if you can.  Error reporting is now improved; the underlying `io::Error` is now `Display`ed when there's a problem.

**Comment by josephDunne** [comment]
It is indeed :-) thank you!
